AEPCore/Mocks/PublicTestUtils/TestBase.swift

```diff
@@ -17,8 +17,6 @@ import AEPServices
 @testable import AEPCore
 
 open class TestBase: XCTestCase {
-    /// Use this property to execute code logic in the first run in this test class; this value changes to False after the parent tearDown is executed
-    public private(set) static var isFirstRun: Bool = true
     /// Use this setting to enable debug mode logging in the `TestBase`
     public static var debugEnabled = false
```

Comment:
Could you also check and remove this boolean? If it’s being used somewhere, please make it non-static.